### ISSUE TITLE
pip install git+git isn't working

### ISSUE THREAD

**Issue by mccruz07** [cytomining/pycytominer#253]
Hi, I's trying to install pycytominer though```pip install git+git://github.com/cytomining/pycytominer``` and isn't working for me, 

```
> pip install git+git://github.com/cytomining/pycytominer
Collecting git+git://github.com/cytomining/pycytominer
  Cloning git://github.com/cytomining/pycytominer to /private/var/folders/3z/k52fjt715kj7bxy2pz0fv_340000gq/T/pip-req-build-ps327vd5
  Running command git clone --filter=blob:none --quiet git://github.com/cytomining/pycytominer /private/var/folders/3z/k52fjt715kj7bxy2pz0fv_340000gq/T/pip-req-build-ps327vd5
  fatal: unable to connect to github.com:
  github.com[0: 140.82.114.4]: errno=Operation timed out

  error: subprocess-exited-with-error
```
so I changed to 

```pip install git+https://github.com/cytomining/pycytominer```

And worked!

**Comment by kenibrewer** [comment]
The `pip install git+git://github.com/cytomining/pycytominer` only works for people who have setup an ssh key connection to github. So even though both options technically work, it probably would be better if the repo documentation included the more general command.

**Comment by kenibrewer** [comment]
This issue is resolved in #289 by @gwaybio

**Comment by kenibrewer** [comment]
@gwaybio Could you close this issue?
